pasystem/pasystem-tool/tool/src/webapp/stylesheets/pasystem.css

```diff
@@ -149,6 +149,7 @@
     max-height: 85%;
     padding: 40px;
     padding-bottom: 0;
+    background-color: var(--sakai-background-color-1) !important;
```

Comment:
@austin48 can you find a selector that is specific enough for this to work without using !important?

Reply:
@ern , I'm using the same selector, but removed the !important, and instead changed the loading order of the pasystem.css file